tests/testthat/test-ace-pipeline.R

```diff
@@ -24,6 +24,13 @@ test_that("ACE loading errors when expected", {
                "not an allowed setting of data_type")
 })
 
+test_that("loading a file with all practice trials does not error, and returns an empty data frame", {
+  result <- load_ace_bulk(aceR_sample_data_path("nexus"), pattern = "all_practice_backwardsspatialspan", data_type="nexus")
+  all_practice_file_path = file.path(aceR_sample_data_path("nexus"), "all_practice_backwardsspatialspan.csv")
```

Comment:
Forgot to add the suggestion block to allow inline committing directly from the review:
```suggestion
```